www/src/pages/404.js

```diff
@@ -14,6 +15,6 @@ export default function FourOhFour() {
         </Link>
       </Container>
       <FooterLinks />
-    </>
+    </Main>
```

Comment:
`<FooterLinks>` should not be nested in `<main>`: https://stackoverflow.com/questions/20470345/should-the-header-and-footer-tags-be-inside-the-main-tag

In general, the components should probably be right next to each other.